editor/components/rich-text/index.js

```diff
@@ -889,6 +891,12 @@ export class RichText extends Component {
 						{ formatToolbar }
 					</div>
 				) }
+				{ isSelected &&
+					<TokenUI
```

Comment:
Would a better name be `InsertableRichTextTokens`, `InsertableInlineTokens`?

Reply:
I'm also a bit torn on the module name `./tokens/ui`. When reviewing this diff it took me longer to distinguish the pieces. Would something more explicit like `./tokens/inserter-fill` be clearer?

Reply:
Yeah, makes sense.

Reply:
Actually `inserter-fill` is not the only thing it's doing. We can move it in a folder though.